app/Main.hs

```diff
@@ -70,7 +70,7 @@ main = do
             , ("Class C", [88, 92, 95, 90, 93, 89, 91, 94, 96, 87, 90, 92])
             , ("Class D", [65, 70, 72, 68, 75, 80, 73, 71, 69, 74, 77, 76])
             ]
-            defPlot{plotTitle = "Test Score Distribution by Class"}
+            defPlot{plotTitle = "Test Score Distribution by Class", xNumTicks = 4}
```

Comment:
<img width="624" height="601" alt="Image" src="https://github.com/user-attachments/assets/f78329c9-deff-4b39-86d1-1e710a1e9ae7" />
Also flipped the min/max characters below..

Reply:
Yeah. I don't like the box plots. They are still here more as a gimmick.